app/experimenter/nimbus-ui/src/components/DirectoryTable/index.scss

```diff
@@ -0,0 +1,7 @@
+.directory-table th {
+  font-weight: normal;
+  border-top: 0;
+  &:first-child {
+    font-weight: bold;
+  }
+}
```

Comment:
Bootstrap has utilities classes that can help with this sort of thing to avoid writing custom styles. So you could give the `th` elements something like

```
className=`border-top-0 ${idx === 0 ? 'font-weight-bold' : 'font-weight-normal'}`}
```

Reply:
nice!